website/source/layouts/guides.erb

```diff
@@ -50,6 +50,10 @@
         </ul>
       </li>
 
+      <li<%= sidebar_current("guides-tls") %>>
+        <a href="/guides/tls.html">Configuring TLS</a>
```

Comment:
@shantanugadgil Thanks!

@sethvargo Renamed and merging! If you get a chance to peek at it again let me know if my changes look good. Thanks!